### PR TITLE
Containerized Docker Application Lifecycle - 06.4

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Guide <-> e-book syncing - Chapter 6 - Article 4

### CODE REVIEW

File: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md [hunk 1]
```diff
@@ -19,77 +19,57 @@ As it pertains to Docker, you can chart life-cycle events and performance counte
...
+There are many support chores with lots of details that you have to handle to ensure your applications and infrastructure are in top notch condition to support business needs, and the situation gets heavier in the microservices realm, so you need a way to have both high-level and detailed views when you need to take action.
 
-### [Operations Management Suite](https://microsoft.com/oms) Container Solution for Docker
+Azure has the tools to manage both your cloud and on-premises resources in a unified view if four critical aspects:
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md]:
```suggestion
Azure has the tools to manage an provide a unified view of four critical aspects of both your cloud and on-premises resources:
```

File: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md [hunk 2]
```diff
@@ -19,77 +19,57 @@ As it pertains to Docker, you can chart life-cycle events and performance counte
...
-![](./media/image2.png)
+- **Backup**. With [Azure Backup](https://azure.microsoft.com/services/backup/).
+  - Avoid costly business disruptions, meet compliance goals, and protect your data against ransomware and human errors.
+  - Keep your backup data encrypted in transit and at rest.
+  - Ensure access based on multifactor authentication to prevent unauthorized use.
 
-Figure 6-2: Information about Docker containers shown by Operations Management Suite
+- **Monitoring**. With [Azure monitoring](https://azure.microsoft.com/solutions/monitoring/), [Log Analytics](https://azure.microsoft.com/services/log-analytics/), and [Application Insights](https://azure.microsoft.com/services/application-insights/).
+  - Get full visibility into the health and performance of your Azure workloads, apps, and infrastructure.
+  - Collect data from any source and get rich insights for your virtual machines, containers, and applications.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md]:
```suggestion
  - Collect data from any source and get rich insights into your virtual machines, containers, and applications.
```

**Review by rpetrusha — APPROVED** [review]
LGTM, @mvelosop. I've left a few suggestions. You can merge when you're ready.

File: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md (hunk 1 above)
Comment (mvelosop) [reply docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md]: That makes an important difference!

**Comment by mvelosop** [comment]
@rpetrusha, this PR seems to be "hanging" could you take a look please?

**Comment by mairaw** [comment]
Closed and reopened to see if build finishes this time

**Comment by mvelosop** [comment]
Hi @mairaw, @rpetrusha, this build doesn't finish, is there some way to find out what's happening?

**Comment by rpetrusha** [comment]
The build succeeded (twice), but for some reason it's not reported here. Closing and reopening once again.

**Comment by rpetrusha** [comment]
Closing and reopening one more time...

**Comment by mvelosop** [comment]
Hi @rpetrusha, @mairaw, I just created PR https://github.com/dotnet/docs/pull/10776 to replace this one, by just copying and pasting over the latest version, and this time the changes went through, only missing the approval! 😊

**Comment by rpetrusha** [comment]
Thanks for doing that, @mvelosop. I have no idea initially why successful builds were not reported, nad then why it wouldn't build at all.

**Pull request closed without merging**